azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -84,21 +84,25 @@ class MQTTTransport(object):
     :type on_mqtt_connection_failure_handler: Function
     """
 
-    def __init__(self, client_id, hostname, username, ca_cert=None, x509_cert=None):
+    def __init__(
+        self, client_id, hostname, username, ca_cert=None, x509_cert=None, websockets=False
+    ):
         """
         Constructor to instantiate an MQTT protocol wrapper.
         :param str client_id: The id of the client connecting to the broker.
         :param str hostname: Hostname or IP address of the remote broker.
         :param str username: Username for login to the remote broker.
         :param str ca_cert: Certificate which can be used to validate a server-side TLS connection (optional).
         :param x509_cert: Certificate which can be used to authenticate connection to a server in lieu of a password (optional).
+        :param websockets: Boolean value to determine whether or not to enable a websockets connection in the Transport.
```

Comment:
>Boolean valu [](start = 27, length = 12)

You can identify the type in the markup with :param bool websockets: rather than having to specify it in the description